mockns1/account_activity.go

```diff
@@ -0,0 +1,19 @@
+package mockns1
+
+import (
+	"net/http"
+
+	"gopkg.in/ns1/ns1-go.v2/rest/model/account"
+)
+
+// AddActivityListTestCase sets up a test case for the api.Client.Activity.List()
+// function
+func (s *Service) AddActivityListTestCase(
+	requestHeaders, responseHeaders http.Header,
+	response []*account.Activity,
+) error {
+	return s.AddTestCase(
+		http.MethodGet, "/account/activity", http.StatusOK, requestHeaders,
+		responseHeaders, "", response,
+	)
+}
```

Comment:
I don't think this is referenced anywhere, maybe add a basic test for the GET?

Reply:
Test file has been added, I'm going to add another run to the suite `>1` parameter as well, to help test fix for https://github.com/ns1/ns1-go/pull/234#discussion_r1552288381 as well